Documents/MobileAppActors.txt

```diff
@@ -0,0 +1,17 @@
+Actors: Regular User, Artist, Admin
+
+Regular User - Draw on Digital Prosthetic, Browse Gallery, Contact Artists
+A regular user would use this app for fun, to enhance their creativity by drawing on the flattened surface of a prosthetic, or simply to browse the galleries without actually visiting the website.
+This would be a fun experience and ensure that the galleries would have a higher visibilty by being exposed to more individuals through the app.
+Amputees would fall under this category as well.
+An amputee might not necessarily use this app to look for prosthetics or artists, they could use this for fun as well, however, artworks could have their author's contact details displayed in the app, so
+an amputee would know who to contact in case they're interested in prosthetic. This could be their main platform (along with the website) for finding unique prosthetics made by professional artists.
```

Comment:
What happens when the interest is taken? What is the next step? Do we link to the website and proceed from there or does mobile also have a submission forms for donations/requests?

Reply:
Probably not, because the point of the app is raising awareness. The main logic center would be in the website.

Reply:
So the link to the website should be there somewhere.

Reply:
I've moved all administrative logic to the website.